extensions/amp-form/0.1/test/integration/test-integration-form.js

```diff
@@ -145,7 +147,9 @@ describes.realWin('AmpForm Integration', {
     });
   });
 
-  describeChrome.run('Submit xhr-POST', () => {
+  describeChrome.run('Submit xhr-POST', function() {
+    this.timeout(RENDER_TIMEOUT);
+
     it('should submit and render success', () => {
```

Comment:
[Arrow functions don't bind `this`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#No_binding_of_this), so we need to use regular functions OR chain a `.timeout()` to the test function.